src/VCEnvironment.cmake

```diff
@@ -4,6 +4,13 @@ include("${ProjectOptions_SRC_DIR}/Utilities.cmake")
 
 # Run vcvarsall.bat and set CMake environment variables
 function(run_vcvarsall)
+  # If MSVC is being used, and ASAN is enabled, we need to set the debugger environment
+  # so that it behaves well with MSVC's debugger, and we can run the target from visual studio
+  if(MSVC)
+    get_all_targets(all_targets)
+    set_target_properties(${all_targets} PROPERTIES VS_DEBUGGER_ENVIRONMENT "PATH=$(VC_ExecutablePath_x64);%PATH%")
```

Comment:
VC_ExecutablePath_x64 seems to be architecture dependant.

Reply:
Fixed in https://github.com/aminya/project_options/pull/121/commits/ea5d6d0a3bb4ff8e6f82451c11440ac9aa2a5dab